app-antivirus/fangfrisch/files/fangfrisch-has-news.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
```

Comment:
Should use the absolute path to bash on gentoo machines, usually `/bin/bash` exactly -- but EPREFIX is relevant here as well.

Reply:
The script was written to be portable, and "#!/usr/bin/env bash" is the correct way to do that. I don't understand the EPREFIX reference, could you elaborate?

Reply:
It's a problem because users may have a different version of bash in $PATH that comes before the system path (/bin) and then you are using something you didn't expect.

You cannot guarantee that such bash executables are new enough to support the features you use, whereas you can guarantee that the version in /bin/bash is managed by Gentoo and is new enough.

EPREFIX is for Gentoo Prefix, e.g. installing Gentoo packages on macOS. See prefix.eclass for example details.

In general this is why shebangs should be configure-time choices handled by build systems. "Just use /usr/bin/env" is somewhat adequate if you're copying scripts around from machine to machine without a package manager and aren't making any assumptions to begin with but a package manager can very much do better.

Reply:
If the user wants to use a different PATH setup, that's their prerogative. It is actually one of the reasons one should use the `#!/usr/bin/env ...` shebang. Also, as I mentioned, this scripts is not written for Gentoo. It is a thoroughly tested, portable example script, which will not run without user modification anyway, and I don't plan on making changes to it. I am glad to fix any mistakes I may have made in the ebuild, of course.